migration-tools/utils.py

```diff
@@ -163,6 +163,16 @@ def migrate_docublock_output(exampleName):
 
     return output
 
+def migrateIndentedCodeblocks(paragraph):
+    indentedCodeblocks = re.findall(r"^\n(?:\s{4}[a-zA-Z].*\n)+^\n", paragraph, re.MULTILINE)
+    for codeblock in indentedCodeblocks:
+        newCodeblock = f"```\n{codeblock}```\n"
+        newCodeblock = re.sub(r"^\s*$\n|^\s*", "", newCodeblock, 0, re.MULTILINE)
```

Comment:
This seems to strip all leading whitespace. I'm not sure if there are any examples with additional whitespace that should be preserved, but it would generally be safer to remove `[ ]{4}` only.